src/Integration/Binding/BindingWorkflow.cs

```diff
@@ -133,7 +137,13 @@ private ProgressStepDefinition[] CreateWorkflowSteps(IProgressController control
                         (token, notifications) => this.PromptSaveSolutionIfDirty(controller, token)),
 
                 new ProgressStepDefinition(Strings.BindingProjectsDisplayMessage, StepAttributes.BackgroundThread,
-                        (token, notifications) => this.DownloadQualityProfile(controller, token, notifications, languages)),
+                        (token, notifications) => this.DownloadBindingParameters(controller, token, notifications)),
+
+                new ProgressStepDefinition(Strings.BindingProjectsDisplayMessage, StepAttributes.Indeterminate,
+                        (token, notifications) => this.DiscoverProjects(controller, notifications)),
```

Comment:
Must be called on the UI thread because I'm accessing the build properties via `IVsBuildPropertyStorage`.